examples/02_control_flow/03_Ranges.md

```diff
@@ -57,12 +57,12 @@ fun main() {
 //sampleStart
     val x = 2
     if (x in 1..10) {            // 1
-        print(x)
+        println("if x is in 1 - 10 range " + "X is "+x)
     }
-    print(" ")
-
-    if (x !in 1..4) {            // 2
-        print(x)
+   
+     val y = 5
+    if (y !in 1..4) {            // 2
+        println("if y is not in 1- 4 range " + "Y is "+y)
```

Comment:
```suggestion
        println("$x is not in range from 5 to 10")
```

Reply:
Please fix the condition of `if` in accordance with the message you print and remove the declaration of `y`.

Reply:
println("$x is not in range from 5 to 10")